### PR TITLE
Pre caching config

### PR DESCRIPTION
**Pull request by anshumansworld**
This includes downloading of config file in init method of Pubnative class, as well as updates in Demo app.

### CODE REVIEW

File: mediation/src/main/java/net/pubnative/mediation/Pubnative.java [hunk 1]
```diff
@@ -27,7 +27,7 @@ public static void init(Context context, String appToken, HashMap<String, String
         } else if (TextUtils.isEmpty(appToken)) {
             Log.v(TAG, "init - warning: invalid apptoken");
         } else {
-            // download or update config
+            // cache config
             PubnativeConfigManager.getConfig(context, appToken, requestParams);
```
Comment (cerberillo) [inline mediation/src/main/java/net/pubnative/mediation/Pubnative.java]: where is the listener?

File: mediation/src/main/java/net/pubnative/mediation/config/PubnativeConfigManager.java [hunk 2]
```diff
@@ -119,6 +119,30 @@ public synchronized static void getConfig(Context context, String appToken, Map
...
+        } else {
+            PubnativeConfigRequestModel item = new PubnativeConfigRequestModel();
+            item.context = context;
+            item.appToken = appToken;
+            item.extras = extras;
+            item.listener = null;
+            enqueueRequest(item);
+            doNextConfigRequest();
+        }
+    }
```
Comment (cerberillo) [inline mediation/src/main/java/net/pubnative/mediation/config/PubnativeConfigManager.java]: I seriously don't see the sense of DUPLICATING the same method.

Please remove this method and reuse the existing one by placing a listener that does NOTHING when it gets callback

**Comment by RagnarR** [comment]
Looks good for me 👍

**Comment by cerberillo** [comment]
Finally it looks nice

**Comment by RagnarR** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;6kWKZHyx&#x2F;6-test-card"> Test card</a>

**Comment by RagnarR** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;6kWKZHyx&#x2F;6-test-card"> Test card</a>

**Comment by cerberillo** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;ftYJGyNN&#x2F;1220-1-fix-config-cache-behaviour-2"> (1) Fix config cache behaviour [2]</a>

**Pull request merged**